extensions/amp-script/validator-amp-script.protoascii

```diff
@@ -85,3 +85,12 @@ tags: {  # <amp-script>
     supported_layouts: RESPONSIVE
   }
 }
+# CANVAS support.
+tags: {
+  html_format: AMP
+  tag_name: "CANVAS"
```

Comment:
Since this is a standard HTML element, would you mind moving it to validator-main.protoascii under the appropriate HTML spec header?

You can leave the mandatory_ancestor thing here, and leave a comment describing where to look.